testsuite/features/core/srv_create_repository.feature

```diff
@@ -50,40 +50,61 @@ Feature: Add a repository to a channel
     And I wait until the channel "test-channel-x86_64" has been synced
     And I disable source package syncing
 
+  Scenario: Add the repository to the x86_64 child channel
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Test-Channel-x86_64 Child Channel"
```

Comment:
Double check if "Test-Channel-x86_64 Child Channel" is the name we have now, I would say that is changed now by "Fake-RPM-SLES15SP4-Channel" right? It's hard to follow it, maybe we need a diagram of the current implementation, similar to what I did as first diagram draft but with the current values for everything. (We could even add the diagram as part of our documentation inside the testsuite folder)

Reply:
In that case this specific child channel is not needed anymore as the `Fake-RPM-SLES15SP4-Channel` one is itself a child channel.
Following Michael's findings I reviewed the need of these steps and removed them in commits to come.
The last references to this child channel are in https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/core/srv_channels_add.feature#L24, wondering if we should plainly remove this, refactor the feature so it deletes the child channel or just leave it be.